project/ProjectPlugin.scala

```diff
@@ -65,19 +64,14 @@ object ProjectPlugin extends AutoPlugin {
       MonadicFor,
       Silencer,
       scalacOptions ++= Seq(
-        "-language:_",
         "-unchecked",
         "-explaintypes",
-        "-Ywarn-value-discard",
```

Comment:
I like this one too...

Reply:
these are already there, we have `sbt-sensible` :wink:

Reply:
💯